lib/control/correlation.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const UUID = require('node-libuuid');
+
+exports.create = function create(options) {
+  options = Object.assign({}, options);
+
+  if (!options.hasOwnProperty('header')) {
+    throw ReferenceError('Missing required parameter `header`!');
+  }
+
+  Log.info(`Using Correlation-Identifier ${options.header}`);
+  const header = options.header;
+
+  return function handle(req, res, next) {
+    if (!req.headers.hasOwnProperty(header)) {
+      req.identifier = req.headers[header] = UUID.v4();
+      Log.debug(`Setting Correlation-Identifier header ${header}:${req.identifier}`, {
+        identifier: req.identifier
+      });
+
+      return next();
```

Comment:
Is it worth capturing this behavior of creating a correlation ID if one's not provided in a test?